RevenueCat/Scripts/PurchaserInfo.cs

```diff
@@ -52,6 +55,8 @@ public PurchaserInfo(JSONNode response)
             FirstSeen = FromUnixTime(response["firstSeenMillis"].AsLong);
             OriginalAppUserId = response["originalAppUserId"];
             RequestDate = FromUnixTime(response["requestDateMillis"].AsLong);
+            OriginalPurchaseDate = FromUnixTime(response["originalPurchaseDateMillis"].AsLong);
```

Comment:
Since `originalPurchaseDateMillis` can be null, you should do something similar to `LatestExpirationDate`. Otherwise it will initialize an empty `DateTime` if `response["originalPurchaseDateMillis"]` is null. At least that's how it worked before I changed the JSON parser. You should double check.